src/NuGetGallery/ViewModels/PackageListViewModel.cs

```diff
@@ -60,5 +64,13 @@ public class PackageListViewModel
         public bool IncludePrerelease { get; }
 
         public bool IsPreviewSearch { get; }
+
+        public string PackageType { get; set; }
+
+        public string SortBy { get; set; }
+
+        public bool shouldDisplayAdvancedSearchPanel { get; set; }
```

Comment:
Nit: C# style is to uppercase all the things

```suggestion
        public bool ShouldDisplayAdvancedSearchPanel { get; set; }
```